src/commands/prompt/index.ts

```diff
@@ -59,6 +65,11 @@ export const command: CommandModule<{}, PromptOptions> = {
         type: 'boolean',
         describe:
           'Forces color output (even if stdout is not a terminal). Use --no-color to disable colors.',
+      })
+      .option('file', {
+        alias: 'f',
```

Comment:
i would leave alias out for now, we can always add it later.
```suggestion
```

Reply:
Done